### PR TITLE
Library Documentation

### PR DESCRIPTION
**Pull request by lkiesow**
This patch extends the library documentation by…

- switching from Markdown to reStructuredText for the readme
- adding a short installation and usage guide to the readme
- properly including the readme into the documentation

Screenshot from 2019-10-11 23-24-48

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -7,14 +7,16 @@ python:
   - "3.6"
   - "3.7"
 
+addons:
```
Comment (tibroc) [inline .travis.yml]: What are the advantages of installing dependencies like this?

File: readme.rst [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    :alt: GitHub issues
+.. image:: https://img.shields.io/github/license/emissions-api/sentinel5dl
+    :target: https://github.com/emissions-api/sentinel5dl/blob/master/LICENSE
+    :alt: MIT license
+
+This library provides easy access to data from the European Space Agency's
+Sentinel 5P satellite.
+
+- `Sentinel5dl on PyPI <https://pypi.org/project/sentinel5dl/>`_
+- `Online documentation <https://sentinel5dl.emissions-api.org>`_
```
Comment (tibroc) [inline readme.rst]: Why this link, here? Isn't this a reference on this page?

File: .travis.yml (hunk 1 above)
Comment (lkiesow) [reply .travis.yml]: Less manual steps and iirc an implied `apt-get update`

File: readme.rst (hunk 2 above)
Comment (lkiesow) [reply readme.rst]: You do not only show this on sentinel5dl.emissions-api.org but also on GitHub and soon on PyPI.

**Pull request merged**